docs/platform/release-notes/tizen-5-5-m3.md

```diff
@@ -0,0 +1,93 @@
+﻿# Tizen 5.5 Public M3 Release Notes
+
+Release date: Aug. 31, 2020
+
+
+- Tizen 5.5 public M3 adopts some new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.
+
+

```

Comment:
Need to add Release Details, see previous release notes.

Reply:
We haven't used "## Release Details" before. I think someone added this heading(## Release Details) regardless of the result of the previous review request.

Reply:
@wtprince, @mijin85cho 

Thanks for clarifying.